composer.json

```diff
@@ -52,7 +52,10 @@
         "drupal/pathauto": "^1.11",
         "drupal/redirect": "^1.8",
         "drupal/seckit": "^2.0",
-        "drupal/social_auth_hid": "^3.2",
+        "drupal/social_auth_google": "^4",
+        "drupal/social_auth_hid": "^5",
+        "drupal/social_auth_linkedin": "^4",
+        "drupal/social_auth_microsoft": "^4",
```

Comment:
Is that still used? The config was removed in https://github.com/UN-OCHA/rwint9-site/pull/1145/commits/73ebd0be4fae2903047124b0e597913dfacdf383